lib/topology/platforms/shell.py

```diff
@@ -259,6 +268,8 @@ class PExpectShell(BaseShell):
     :param bool try_filter_echo: On platforms that doesn't support some way of
      turning off the echo of the command try to filter the echo from the output
      by removing the first line of the output if it match the command.
+    :param auto_connect bool: Enable the automatic creation of a connection
+     when ``send_command`` is called if not default exists.
```

Comment:
This is wrong. The implemented behavior does not depend on the existence of the default connection.